src/Gui/ApplicationPy.cpp

```diff
@@ -777,6 +778,15 @@ PyObject* ApplicationPy::sExport(PyObject * /*self*/, PyObject *args)
                     printer.setPdfVersion(QPagedPaintDevice::PdfVersion_A1b);
                     printer.setOutputFormat(QPrinter::PdfFormat);
                     printer.setOutputFileName(fileName);
+
+                    QString appname = QCoreApplication::applicationName();
+                    auto config = App::Application::Config();
+                    QString major = QString::fromUtf8(config["BuildVersionMajor"].c_str());
+                    QString minor = QString::fromUtf8(config["BuildVersionMinor"].c_str());
+                    QString point = QString::fromUtf8(config["BuildVersionPoint"].c_str());
+                    QString suffix = QString::fromUtf8(config["BuildVersionSuffix"].c_str());

```

Comment:
I'd just use `fromStdString` here and let Qt deal with the internal mechanics. Same below.